css/craft.css

```diff
@@ -315,6 +315,22 @@ body {
     font-family: Roboto, serif;
 }
 
+.ore-item .hovertext, .queue-item .hovertext, .cft-item .hovertext {
+    font-family: Roboto, serif;
+    visibility: hidden;
+    text-align: left;
+    border-radius: 6px;
+    padding: 10px 10px;
+
+    position:absolute;
+    top: 20px;
```

Comment:
If removed, tooltip shows where it should... as long as parent list isn't scrolled down, in which case tooltip shifts way lower / below screen edge